robot/Cumulus/tests/browser/gift_entry/review_donations_create_opp_sge.robot

```diff
@@ -0,0 +1,72 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             robot/Cumulus/resources/PaymentPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Enable Gift Entry
+...             Setup Test Data
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Keywords ***
+Setup Test Data
+    &{account} =         API Create Organization Account    Name=${faker.company()}
+    Set suite variable   &{account}
+    ${date} =            Get Current Date         result_format=%Y-%m-%d
+    Set suite variable   ${date}
+    &{opportunity} =     API Create Opportunity   ${account}[Id]              Donation  
```

Comment:
Since these are global variables being defined, it is advisable to name them in all caps.

Reply:
updated